ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/participant/ParticipantTestContext.scala

```diff
@@ -459,4 +466,41 @@ private[testtool] final class ParticipantTestContext private[participant] (
   def submitAndWaitForTransactionTree(request: SubmitAndWaitRequest): Future[TransactionTree] =
     services.command.submitAndWaitForTransactionTree(request).map(_.getTransaction)
 
+  /** Moves all time values in the request (the LET and MRT) by the specified amount.
+    * This simulates a request from a client with a skewed clock. */
+  def moveRequestTime(
```

Comment:
Do you think this is useful enough to have in the participant context, or should this rather be a utility function within the test?